vocode/streaming/vector_db/pinecone.py

```diff
@@ -0,0 +1,33 @@
+import os
+import asyncio
+from functools import partial
+from langchain.vectorstores import Pinecone
+from vocode.streaming.models.vector_db import PineconeConfig
+from langchain.embeddings.openai import OpenAIEmbeddings
+from vocode.streaming.vector_db.base_vector_db import VectorDB
+
+
+class PineconeDB(VectorDB):
+    def __init__(self, config: PineconeConfig) -> None:
+        import pinecone
+
+        self.config = config
+        self.pinecone = pinecone
```

Comment:
why reassign the import?

Reply:
Removed. Left over from previous implementation.